File: repos_estudiantes_TARQUI_segunda_entrega/Quispe_Huanca_Carlos_Daniel/HERENCIA/EJERCICIO-3/Persona/src/ejercicio3/Estudiante.java
package ejercicio3;

public class Estudiante extends Persona {
    private String ru;
    private String fechaIngreso;
    private int semestre;
    
    public Estudiante() {
        this("", "", "", "", "2000-01-01", "", "", "2020-01-01", 1);
    }
    
    public Estudiante(String ci, String nombre, String apellido, String celular, String fechaNac, String sexo,
                     String ru, String fechaIngreso, int semestre) {
        super(ci, nombre, apellido, celular, fechaNac, sexo);
        this.ru = ru;
        this.fechaIngreso = fechaIngreso;
        this.semestre = semestre;
    }
    
    @Override
    public void mostrar() {
        super.mostrar();
        System.out.println("RU: " + ru);
        System.out.println("Fecha Ingreso: " + fechaIngreso);
        System.out.println("Semestre: " + semestre);
    }
    
    public String getRu() { return ru; }
    public String getFechaIngreso() { return fechaIngreso; }
    public int getSemestre() { return semestre; }
    
    public void setRu(String ru) { this.ru = ru; }
    public void setFechaIngreso(String fechaIngreso) { this.fechaIngreso = fechaIngreso; }
    public void setSemestre(int semestre) { this.semestre = semestre; }
}

File: repos_estudiantes_TARQUI_segunda_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/HERENCIA/EJERCICIO-3/Estudiante.py
from Persona import Persona

class Estudiante(Persona):
    def __init__(self, ci="", nombre="", apellido="", celular="", fecha_Nac="", ru="", fecha_Ingreso="", semestre=0):
        super().__init__(ci, nombre, apellido, celular, fecha_Nac)
        self.ru = ru
        self.fecha_Ingreso = fecha_Ingreso
        self.semestre = semestre

    def mostrar(self):
        super().mostrar()
        print(f"RU: {self.ru}, Fecha Ingreso: {self.fecha_Ingreso}, Semestre: {self.semestre}")
